docs/integration/authorization.adoc

```diff
@@ -17,7 +17,7 @@ As the authorization process – in difference to the rest of the communication
 
 The authorization process is major for farming software and telemetry platforms, for CUs it is an optional function.
 
-For CUs, this requires additional infrastructure, therefore DKE advices to use the TAN exchange process.
+For CUs, this requires additional infrastructure, therefore DKE advices to use the Registration Code exchange process.
```

Comment:
From my point of view - "Registration Code" should be written as "registration code"

Reply:
adjusted